package.json

```diff
@@ -40,19 +32,30 @@
     "./es": "./es/",
     "./lib": "./lib"
   },
+  "main": "lib/index.js",
+  "unpkg": "dist/ramda-land.js",
+  "module": "es/index.js",
+  "types": "index.d.ts",
+  "files": [
+    "es",
+    "lib",
+    "dist",
+    "index.d.ts"
+  ],
   "scripts": {
-    "clean": "rimraf es lib dist",
-    "build:es": "cross-env BABEL_ENV=es babel --ignore '**/*.test.js' index.js src/ --out-dir es",
-    "build:cjs": "cross-env BABEL_ENV=commonjs babel --ignore '**/*.test.js' index.js src --out-dir lib",
-    "build:umd": "rollup -c",
     "build": "yarn clean && yarn build:es && yarn build:cjs && yarn build:umd",
-    "prepack": "yarn build",
+    "build:cjs": "cross-env BABEL_ENV=commonjs babel --ignore '**/*.test.js' src --out-dir lib",
+    "build:es": "cross-env BABEL_ENV=es babel --ignore '**/*.test.js' src/ --out-dir es",
+    "build:umd": "rollup -c",
+    "clean": "rimraf es lib dist",
     "docs": "jsdoc -c jsdoc.json",
-    "lint": "eslint index.js src/",
+    "lint": "eslint src/",
+    "prepack": "yarn build && yarn types",
     "prettier": "prettier --write src/ index.js",
     "test": "jest --env=node --colors --bail --detectOpenHandles --errorOnDeprecated --noStackTrace src/",
     "test:coverage": "yarn test --coverage --coverageDirectory .coverage",
-    "test:watch": "yarn test --watchAll"
+    "test:watch": "yarn test --watchAll",
+    "types": "jsdoc -t node_modules/tsd-jsdoc/dist -r src -d . && move-file types.d.ts index.d.ts && replace-in-files --regex='declare\\s+function' --replacement='export function' index.d.ts && replace-in-files --regex='^' --replacement='/*\\n  This file was auto-generated by tsd-jsdoc (https://github.com/englercj/tsd-jsdoc)\\n*/\\n\\n' index.d.ts"
```

Comment:
This is a bit scary, I know - what was the most succinct way I could put together to generate an `index.d.ts` file with a banner at the top and the appropriate `export` declarations, preserving portability (i.e.: no `sed`, no `awk`).